package.json

```diff
@@ -67,6 +68,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-helmet": "^6.1.2",
     "concurrently": "^6.2.1",
```

Comment:
Added react-helmet to add each page meta data

Reply:
Actually replace it with `react-helmet-async` cuz it was giving a warning, which was using the deprecated componentWillMount
https://github.com/ice-cream-the-fat-cat/my-niwa-frontend/pull/143/commits/2c8327e683143e220a40b05a05015f02dd4ee88a